app/views/expenses/index.json.jbuilder

```diff
@@ -1 +1 @@
-json.array! @expenses, partial: "expenses/expense", as: :expense
```

Comment:
Please improve the design of transactions page as per the design guidelines. Kindly follow the same layout, fonts, and colors as defined in the orignal design page.
![transac](https://user-images.githubusercontent.com/35504149/204051583-5cba110d-b5b1-4215-9c88-f987fc5c04ed.png)